src/base.less

```diff
@@ -0,0 +1,336 @@
+// Official palette.
+// Current as of November 13, 2015.
```

Comment:
This is a little out of date, now. https://github.com/cfpb/design-manual/blob/gh-pages/src/static/css/brand-palette.less is the latest version. Sorry for the confusion!